docs/guides/references/_changelogs/12.4.0.md

```diff
@@ -0,0 +1,49 @@
+## 12.4.0
+
+_Released 1/24/2023_
+
+**Features:**
+
+- The `cy.log()`command will now display a line break where the `\n` character
+  is used. Addressed in
+  [#25467](https://github.com/cypress-io/cypress/pull/25467).
+- `.jsx`/`.tsx` file extensions will be utilized when generating a new empty
+  spec file inside component testing if the project contains at least one file
+  with those extensions. Addressed in
+  [#25318](https://github.com/cypress-io/cypress/pull/25318).
+- Added new `experimentalSkipDomainInjection` flag to disable Cypress from
+  setting `document.domain` on injection. Addressed in
+  [#25307](https://github.com/cypress-io/cypress/pull/25307).
+- Added new `experimentalMemoryManagement` flag to improve memory management in
+  Chromium-based browsers. Addressed in
+  [#25462](https://github.com/cypress-io/cypress/pull/25462).
+- Video output link in `cypress run` mode has been added to it's own line to
```

Comment:
I'd list this as misc and say this was for readability because we can't guarantee this will be clickable in all terminals.

Reply:
moved to `Misc` in 4ba7d032b